civictechprojects/views.py

```diff
@@ -144,8 +155,9 @@ def projects_list(request):
         query_params = urlparse.parse_qs(
             url_parts, keep_blank_values=0, strict_parsing=0)
         projects = (
-            projects_by_keyword(query_params)
-            | projects_by_tag(query_params)
+            projects_by_description(query_params) or
+            projects_by_name(query_params) or
+            projects_by_tag(query_params)
```

Comment:
Does the 'or' operator combine the results of these queries, or does it just take the first result that is non-empty?

Reply:
The 'or' does not combine the results of the query. The and/or logic is the next thing that I'll be working on.

I put the 'or's in as a placeholder and pushed this change so that the 'keyword' search would not only search the description of a project, but also it's name.